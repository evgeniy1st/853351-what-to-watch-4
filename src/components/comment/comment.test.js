import React from "react";
import renderer from "react-test-renderer";
import Comment from "./comment.jsx";

it(`render comment`, () => {
  const comment = {
    rating: 8.9,
    text: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
    author: `Kate Muir`,
    date: new Date(2016, 11, 24),
    id: 45220
  };
  const id = 43388;

  const tree = renderer
    .create(<Comment
      comment={comment}
      id={id}
    />)
    .toJSON;

  expect(tree).toMatchSnapshot();
});
