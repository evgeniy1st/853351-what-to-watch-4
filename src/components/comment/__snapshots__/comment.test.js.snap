// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render comment 1`] = `[Function]`;
